docs/production-deployment/cloud/gcp-export-gcs.mdx

```diff
@@ -0,0 +1,168 @@
+---
+id: gcp-export-gcs
+title: Exporting Workflow Event History to GCS
+sidebar_label: GCP Export to GCS
+description: Export Workflow History to GCS
+slug: /cloud/export/gcp-export-gcs
+toc_max_heading_level: 4
+keywords:
+  - explanation
+  - how-to
+  - operations
+  - temporal cloud
+  - term
+  - verify
+  - workflow history export
+  - gcp
+tags:
+  - explanation
+  - how-to
+  - operations
+  - temporal-cloud
+  - term
+  - verify
+  - workflow-history-export
+  - gcp
+---
+
+## Prerequisites {#prerequisites}
+
+Before configuring the Export sink, please complete the following steps in Google Cloud.
+
+1. Create a GCS bucket and take note of its bucket name, for example, "test-export"
+Enable customer-managed encryption keys (CMEK) if you need additional security for your GCS bucket.
+2. Record the GCP Project ID that owns the bucket.
+Create a service account in the same project that grants Temporal permission to write to your GCS bucket.
```

Comment:
I don't think this shows up as you were expecting:
<img width="969" alt="image" src="https://github.com/user-attachments/assets/c6da6304-d5cc-4211-bfb3-b3dba989c8e9">

I think if you want to align things on a separate line, you need to keep the indentation? And add a blank line to make sure it does not get added at the end of the previous line. Alternatively, we could use a sub-dot ?

```suggestion
1. Create a GCS bucket and take note of its bucket name, for example, "test-export"

   Enable customer-managed encryption keys (CMEK) if you need additional security for your GCS bucket.
   
2. Record the GCP Project ID that owns the bucket.

   Create a service account in the same project that grants Temporal permission to write to your GCS bucket.
```

Or something like that...